dronegen/tag.go

```diff
@@ -179,6 +179,7 @@ func tagPipeline(b buildType) pipeline {
 		"GOPATH":  value{raw: "/go"},
 		"OS":      value{raw: b.os},
 		"ARCH":    value{raw: b.arch},
+		//"WINDOWS_SIGNING_CERTIFICATE: value{fromSecret: "WINDOWS_SIGNING_CERTIFICATE"},
```

Comment:
I realise this probably isn't complete yet, but you should gate this so it only exposes the secret when `b.os == "windows"`.

Reply:
Good catch, I'll make the change

Reply:
I've made the change, but for testing now also pass in the dummy cert via a raw drone env var.